docker/dockerfiles/Dockerfile_celery

```diff
@@ -0,0 +1,32 @@
+FROM dimagi/commcarehq_base:latest
+
+RUN apt-get update
+RUN apt-get install -y netcat
+
+RUN mkdir requirements
+COPY scripts/uninstall-requirements.sh requirements/uninstall-requirements.sh
+COPY requirements/uninstall-requirements.txt requirements/uninstall-requirements.txt
+
+RUN requirements/uninstall-requirements.sh
+
+COPY requirements/requirements.txt /vendor/requirements.txt
+COPY requirements/dev-requirements.txt /vendor/dev-requirements.txt
+COPY requirements/test-requirements.txt /vendor/test-requirements.txt
+COPY requirements/prod-requirements.txt /vendor/prod-requirements.txt
+
+RUN pip install \
+    -r /vendor/requirements.txt \
+    -r /vendor/dev-requirements.txt \
+    -r /vendor/prod-requirements.txt \
+    --user --upgrade \
+    && rm -rf /root/.cache/pip
+
+WORKDIR /mnt
+
+RUN mkdir /mnt/shared
+
+ADD docker/entrypoint.sh /mnt/docker/entrypoint.sh
+
+ENTRYPOINT ["/mnt/docker/entrypoint.sh"]
+CMD ["help"]
```

Comment:
this is almost identical to `Dockerfile_commcarehq`. Is it not worth updating that one rather than building a whole new image?